events/EventQueue.cpp

```diff
@@ -14,10 +14,10 @@
  * limitations under the License.
  */
 #include "events/EventQueue.h"
-
 #include "events/mbed_events.h"
-#include "mbed.h"
 
+using namespace events;
```

Comment:
Please enclose the definitions inside the namespace event. `using` is not meant to be use that way: 

```c++
namespace events { 

// definitions

} // namespace event 
```